tests/kuttl/tests/openstackbaremetalset_scale/11-assert.yaml

```diff
@@ -33,7 +33,4 @@ status:
       hostname: compute-1
   provisioningStatus:
     readyCount: 1
-    reason: Existing BaremetalHost "openshift-worker-0" not found for OpenStackBaremetalSet
-      compute.  Please check BaremetalHost resources and re-add "openshift-worker-0"
-      to continue
```

Comment:
This was unnecessary.  Just checking the OSBMS `status.provisioningStatus.readyCount` and `status.provisioningStatus.state` is enough to detect the problem we are expecting.